skrub/_to_float32.py

```diff
@@ -0,0 +1,213 @@
+import numpy as np
+import pandas as pd
+
+from . import _dataframe as sbd
+from ._dispatch import dispatch
+from ._on_each_column import RejectColumn, SingleColumnTransformer
+
+
+@dispatch
+def _to_float32(col, strict):
```

Comment:
there is also a more generic `to_float32` in `_common` that does a simple cast, but this private helper does something more specific because it adds the strict option and parses strings, so I'm not sure

Reply:
I'd be in favor to have only one of them if the one in `_common` is the same as the one in `_to_float` with default parameters, WDYT ? I think it could be misleading to have 2 of the "same thing" in the codebase